docs/en/writing-running-appium/tutorial/swipe/simple-screen.md

```diff
@@ -0,0 +1,70 @@
+## Simple Screen swipe
+
+Phone applications have interface. Interface, in most cases, blocks your
```

Comment:
`Interface, in most cases, blocks your swipe actions when you start from it`

Sorry, could you please explain what this sentence is trying to say?

Reply:
there is scroll view with elements. when you try to scroll and start from any of these elements all depends on how code written. it may occur that element blocks touch while trying to catch click action. quite often case. and any similar cases

Reply:
Could we put the detailed description above into the doc?

Reply:
it is said in next sentences.